chart/values.schema.json

```
@@ -4148,6 +4148,12 @@
                     "x-docsSection": "Common",
                     "default": false
                 },
+                "replicas": {
+                    "description": "Number of pgbouncer pods to deploy",
+                    "type": "integer",
+                    "x-docsSection": "PgBouncer",
+                    "default": 1,
```

```suggestion
                    "default": 1
```
Static checks and tests are failing. I believe this extra `,` could be the reason making it an invalid json.